src/helpers/figures/charts/bar_chart.ts

```diff
@@ -235,9 +273,9 @@ export function createBarChartRuntime(chart: BarChart, getters: Getters): BarCha
       ...CHART_COMMON_OPTIONS,
       indexAxis: chart.horizontal ? "y" : "x",
       layout: getBarChartLayout(definition),
-      scales: getBarChartScales(definition, chartData),
+      scales: getBarChartScales(getters, definition, chartData),
```

Comment:
Oh, I see! Can update the `definition` with the evaluated axis designs beforehand, 
```ts
  const updatedDefinition: GenericDefinition<BarChartDefinition> = {
    ...definition,
    axesDesign: evaluatedAxesDesign,
  };
```

so we don't have to pass it through the function parameter chain like `getChartAxis > getChartAxisTitleRuntime` to  set evaluated value?

Reply:
You should put the evaluated title inside `ChartRuntimeGenerationArgs` rather than updating the definition on the fly. That's where we now put the data extracted from the model (evaluated dataset, evaluated labels, locale, etc). You can add keys for your evaluated title/axis title.

Reply:
Yeah, I’ve already done that. I was taking about `getBarChartScales`, which accepts args which includes `evaluatedAxesDesign` now. We need to pass this to `getChartAxis > getChartAxisTitleRuntime` to set the evaluated value.  

Can we instead create an `updatedDefinition` object that includes the evaluated axesDesign, and then pass it like this:
```ts
    scales.y = getChartAxis(updatedDefinition, "left", "labels", options);
```

Reply:
personnaly, I would  add an argument instead of trying to update the definition on the fly I think